components/gallery/GalleryItem.vue

```diff
@@ -32,8 +32,7 @@
                   <MediaItem
                     :key="resource.src"
                     :src="resource.src"
-                    is-detail
-                    :original="isMobile" />
```

Comment:
how about on mobile?

Reply:
@preschian For square images: there is no difference before and after the PR.

For [non-square images](https://canary.kodadot.xyz/bsx/gallery/4017994657-1):
![CleanShot 2023-05-14 at 21 52 40](https://github.com/kodadot/nft-gallery/assets/128157824/e9c1e094-7395-49a6-8125-3472e7b82daf)

@exezbcz Regarding the changes in non-square images on mobile devices, I guess this is what you intended, as it was part of your requirements for tablet. and this page's mobile and tablet designs are actually the same.

But if you want to keep the original display logic for non-square images on mobile devices, please let me know.

Reply:
Yup, that works. We made a full-screen button for the non-square content, so this is expected. Its a standard of how we display content all over the app, let's keep it the same and consistent. 
Thanks! I will check later.

Reply:
wdyt? Open to discussion :D

Reply:
I think we already had this kind of discussion in the past but I can't find the related issue.

(Looks like buttons are a bit off on mobile screenshot)

Reply:
@exezbcz Another option: we keep the same logic as before for displaying images on mobile and tablet (in their original scale, not square), but remove the annoying extra space as shown below. This is not consistent with the Desktop side, but I can think of an advantage: if the height of the photo is smaller than the width, then it is more space efficient on mobile.

<img width=400 src=https://github.com/kodadot/nft-gallery/assets/128157824/7527ab93-d1c7-4a53-8afd-cb317cd3c565 >